gapic/src/main/com/google/gapid/perfetto/models/GpuInfo.java

```diff
@@ -76,6 +76,7 @@ public int bufferCount() {
       res.forEachRow(($, r) -> {
         switch (r.getString(2)) {
           case "gpu_render_stage":
+          case "vulkan_events":
```

Comment:
We will have custom rendering for this track, so please don't re-use the queue class, which is used only for GPU queue tracks.

Reply:
Hi Pascal, and then is it OK to custom this new track as a SliceTrack.forVulkanEvents(...)?  Is this a right place to put it?

Reply:
Yes, if the slice track model makes sense, we can use it. But I'm thinking that we may need a different model, since we are not really dealing with slices and need to deal with the submission id linking thing.

Reply:
I see, I will go along the way of writing a separate track for vulkan_events, in case in the future there comes more interaction between vulkan_events and other panels. 
There might be some duplicates for this track in the very first draft, but it will make this panel more flexible and benefit the future.

Reply:
I have updated the patch with change in perfetto.  This is just so that you can apply the patch and it should work as is.  

Stella, approve this if you like me to check this in.  But it sounds like you may want to do a different approach.  So feel free to just close this PR if it's not going to help you.

Reply:
Hi Raymond, would do mind only keep the change for perfetto version hash and perfetto settings flag, and have them checked in? I will take care of the other UI bits, thanks!

Reply:
Let's wait for Adi's PR then (https://github.com/google/gapid/pull/3697).  He has updated the hash to be the same as this PR.  I don't want to delay his PR any longer.

Reply:
#3697 has now merged